app/services/CountriesService.scala

```diff
@@ -46,4 +46,11 @@ class CountriesService @Inject() (referenceDataConnector: ReferenceDataConnector
 
   private def sort(countries: Seq[Country]): CountryList =
     CountryList(countries.sortBy(_.description.toLowerCase))
+
+  def doesCountryRequireZip(country: Country)(implicit hc: HeaderCarrier): Future[Boolean] =
+    getCountriesWithoutZip().map(!_.contains(country.code))
+
+  def getCountriesWithoutZip()(implicit hc: HeaderCarrier): Future[Seq[CountryCode]] =
+    referenceDataConnector
+      .getCountriesWithoutZip()
```

Comment:
Are there unit tests for these in departures that can be copied over?

Reply:
@jackheslop96 done